### PR TITLE
Add channels

### LINKED ISSUE #18: Subnetworks
Each Integrated Dynamics component has an ID number. I think it would be useful to add a filter to the Import/Export interfaces to Push/Pull items from a specific Item Interface ID on the network. Could be applied to Energy/Fluid Interfaces as well.

### PR DESCRIPTION
**Pull request by josephcsible**
Closes #18. Needs CyclopsMC/IntegratedDynamics#332

### CODE REVIEW

File: src/main/java/org/cyclops/integratedtunnels/part/aspect/TunnelAspectWriteBuilders.java [hunk 1]
```diff
@@ -77,6 +80,14 @@
 
     public static final IAspectPropertyTypeInstance<ValueTypeBoolean, ValueTypeBoolean.ValueBoolean> PROP_BLACKLIST =
             new AspectPropertyTypeInstance<>(ValueTypes.BOOLEAN, "aspect.aspecttypes.integratedtunnels.boolean.blacklist.name");
+    public static final IAspectPropertyTypeInstance<ValueTypeInteger, ValueTypeInteger.ValueInteger> PROP_CHANNEL =
```
Comment (rubensworks) [inline src/main/java/org/cyclops/integratedtunnels/part/aspect/TunnelAspectWriteBuilders.java:83]: Like in ID, I feel like this should not be needed. If we keep it though, we should remove this anyways and use the property defined in ID instead.

**Review by rubensworks — COMMENTED** [review]
Same comment as the ID PR, so let's discuss the issue there.

**Review by rubensworks — APPROVED** [review]

File: src/main/java/org/cyclops/integratedtunnels/capability/network/TunnelNetworkCapabilityConstructors.java [hunk 2]
```diff
@@ -26,20 +28,22 @@
     @SubscribeEvent
     public void onNetworkLoad(AttachCapabilitiesEventNetwork event) {
         ItemNetwork itemNetwork = new ItemNetwork();
+        IItemChannel itemChannel = itemNetwork.getChannel(IChanneledNetwork.DEFAULT_CHANNEL);
         event.addCapability(new ResourceLocation(Reference.MOD_ID, "itemNetwork"),
                 new DefaultCapabilityProvider<IItemNetwork>(() -> ItemNetworkConfig.CAPABILITY, itemNetwork));
         event.addCapability(new ResourceLocation(Reference.MOD_ID, "itemStorageNetwork"),
-                new DefaultCapabilityProvider<IItemHandler>(() -> CapabilityItemHandler.ITEM_HANDLER_CAPABILITY, itemNetwork));
+                new DefaultCapabilityProvider<IItemHandler>(() -> CapabilityItemHandler.ITEM_HANDLER_CAPABILITY, itemChannel));
```
Comment (josephcsible) [inline src/main/java/org/cyclops/integratedtunnels/capability/network/TunnelNetworkCapabilityConstructors.java:35]: As per https://github.com/CyclopsMC/IntegratedDynamics/pull/332/files#r155936160, do we even need these capabilities for anything?

Comment (rubensworks) [reply src/main/java/org/cyclops/integratedtunnels/capability/network/TunnelNetworkCapabilityConstructors.java:35]: Answered in the referenced comment.

**Pull request merged**